utils/rector/src/RuleRequirementsChecker.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Utils\Rector;
+
+use Composer\InstalledVersions;
+
+final class RuleRequirementsChecker
+{
+    public static function checkRequirements(): void
+    {
+        if (version_compare('0.18', InstalledVersions::getVersion('rector/rector'), '>')) {
```

Comment:
What if I want to upgrade without rector?

Reply:
I'm not sure to understand this comment: this `RuleRequirementsChecker` is defined for rector. It is called from `/utils/rector/config/foundry-set.php` which we can only reach with rector :thinking: But using a version <0.18 will throw an exception

Reply:
Thanks for the explanation 👍